js/ui/notify.js

```diff
@@ -29,4 +39,100 @@ const notify = function(message, /* optional */ type, displayTime) {
     new Toast($notify, options).show();
 };
 
+const createStackContainer = (id) => {
+    const newContainer = $('<div>').appendTo($(domAdapter.getBody()));
+    newContainer.attr('id', id);
+
+    return newContainer;
+};
+
+const getStackContainer = (id) => {
+    if($(`#${id}`).length) {
+        return $(`#${id}`);
+    } else {
+        return createStackContainer(id);
+    }
+};
+
+
+const setContainerClasses = (container, direction) => {
+    container.removeClass();
+    container.addClass('dx-toast-container').addClass(`dx-toast-container-${direction}-direction`);
+};
+
+const setContainerStyles = (container, direction, position) => {
+    const styles = typeof position === 'string'
```

Comment:
let's create `dimensions` object with window and toast dimensions here and pass it to the functions. It can help us to make this calculations only once